charts/selenium-grid/values.yaml

```diff
@@ -375,17 +376,17 @@ chromeNode:
     # failureThreshold: 120
     # periodSeconds: 5
   # Time to wait for pod termination
-  terminationGracePeriodSeconds: 30
+  terminationGracePeriodSeconds: 3600
```

Comment:
Why?

Reply:
If the autoscaler chooses to kill a pod that's running a test currently, we would have at most one hour for the running test to complete rather than 30 seconds. 
If the test doesn't complete within the configured grace period it would result in failure so giving it the maxium allowed time would help reduce these failures.

Reply:
Wouldn't this affect the setup of users who are already having this configuration? Isn't there an alternative to this hard coded value? One of the reasons we added the draining after X sessions feature was to enable this use case, so the pod could live until the session is done.

Reply:
To expand on what @diemol writes: 

Wouldn't it be more straight forward to use ScaledJob combined with draining after one session?
I wrote about it here:

https://github.com/SeleniumHQ/selenium/issues/9845#issuecomment-1206257437

Reply:
@diemol Killing after x sessions wouldn’t help. Consider this scenario where you create two sessions, one runs for 60 mins and another 1 min. As soon as the first is done, the hpa will try to bring down one of the pod irrespective of the configured x sessions. We have no control on which pod is chosen by hpa. This is not current flow but will happen when KEDA scales down.

Also I don’t understand the implication that would be caused by increasing it even for current users. No one would want their tests to be interrupted. Also with this value has no effect unless we have prestop lifecycle hook enabled. Even if that is enabled its gonna delay the pod termination only until the tes session is completed, which i think be desired behaviour

Reply:
@msvticket Jobs doesn’t work great, if you are on severless kubernetes platform like EKS fargate. One new node is created for each pod and everytime it has to download the full browser node image and to create the node in itself combined takes around 90 seconds. Same would the case with GKE autopilot. Also i don’t see a point in killing the pod after just one session. Imagine a organisation of our scale, we need close to 75 pods running during the working hours. Imagine the number of sessions and how many times we would be creating  and killing pods.

The best part about KEDA scaler is that it works for both pods and jobs so if you think jobs would suit your use case better, its just matter of swapping the scaled object resource with scaled jobs. Let me know if you need any assistance, i would be ahppy to help

Reply:
@prashanth-volvocars draining after X sessions will make the container exit and ideally the pod should stop, am I missing something?

Reply:
I'd say that for most users ScaledJobs would be the best fit so this should be the default in the helm chart. Creating a pod is normally not expensive in Kubernetes; Fargate is a special case. But sure, you can skip that and I guess it's me or @diemol that will implement it.

Reply:
@diemol yes, we no longer need X sessions as the pod can handle sessions as long as there are pending sessions in the queue.

Reply:
@msvticket The default should be what works for every environment, not for a specific few. Jobs works for a few, not all, but pod works for all. If you think a scaled job suits your need best, it's available as an option that you can very well enable, but making it default will entirely break it for specific environments. Hope you understand.

Reply:
What option? I don't see any mention of it in the chart.